website/www/site/content/en/blog/review-input-streaming-connectors.md

```diff
@@ -184,15 +184,15 @@ and <a href="https://spark.apache.org/docs/latest/api/python/pyspark.streaming.h
    </td>
    <td>N/A
    </td>
-   <td><a href="https://spark.apache.org/docs/latest/api/python/pyspark.streaming.html#pyspark.streaming.kafka.KafkaUtils">KafkaUtils</a>
+   <td><a href="https://spark.apache.org/docs/latest/streaming-kafka-0-10-integration.html">KafkaUtils</a>
```

Comment:
this is an old blog and the info is outdated. 

@rszper you could perhaps reference older version of pyspark to preserve the original narrative.
 https://spark.apache.org/docs/2.4.8/api/python/pyspark.streaming.html#pyspark.streaming.kafka.KafkaUtils

Reply:
Good idea.